package.json

```diff
@@ -13,6 +13,7 @@
     "preview-local:worker": "npm run build:worker && npm run dev:worker",
     "preview-remote:worker": "npm run build:worker && wrangler versions upload",
     "deploy:worker": "npm run build:worker && wrangler deploy",
+    "tsx": "tsx",
```

Comment:
Seemingly needed to let windows run the tsx pipelines

Reply:
i added a `tsx` command at the bottom for an unrelated reason, so this should hopefully be unnecessary!

Reply:
/ might break the email update script!

Reply:
ty for catching that, the file didn't give any merge conflicts. I guess on account of the space between

Reply:
sg